packages/table/test/selectableTests.tsx

```diff
@@ -31,13 +31,15 @@ describe("DragSelectable", () => {
 
     it("does click selection", () => {
         const onSelection = sinon.spy();
+        const onFocus = sinon.spy();
```

Comment:
onFocus is required much the same way that onSelection is required -- there needs to be something specified here regardless. 

Should probably be testing them though.